.gitpod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-full
+                    
+USER gitpod
+
+# Install custom tools, runtime, etc. using apt-get
+# For example, the command below would install "bastet" - a command line tetris clone:
+#
+# RUN sudo apt-get -q update && #     sudo apt-get install -yq bastet && #     sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Why is this line commented out and not initialized in the way indicated e.g. in https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md#using-the-windows-subsystem-for-linux? Git won't compile out of the box without _some_ dependencies.